contracts/migration/baobab/Aggregator/20231114131186_CHF-USD.json

```diff
@@ -0,0 +1,18 @@
+{
+	"deploy": {
+		"name": "CHF-USD",
+		"timeout": 0,
+		"validator": "0x0000000000000000000000000000000000000000",
+		"decimals": 8,
+		"description": "CHF-USD"
+	},
+	"changeOracles": {
+		"removed": [],
+		"added": [
+			"0xa68a486a654Ca02623e8A41f703A1b24A2cd666e"
+		],
+		"minSubmissionCount": 1,
+		"maxSubmissionCount": 1,
+		"restartDelay": 0
+	}
+}
```

Comment:
@nick-bisonai just small comment, I think you used different tab formatting that our general where we use `tabWidth` with 2 space. Maybe worthy to check it for next time so we follow the same formatting.  :)